nexus/tests/unit/test_simulation_module.py

```diff
@@ -2387,7 +2387,10 @@ def test_execute():
     assert(os.path.exists(outfile))
     assert(os.path.exists(errfile))
     assert(open(outfile,'r').read().strip()=='run')
-    assert(open(errfile,'r').read().strip()=='')
+    err_contents = open(errfile,'r').read().strip()
+    # workaround for openmpi issue on some systems
```

Comment:
Is there more context about this issue, or a link to some more discussion?
It appears to be a very specific problem that apparently doesn't affect actual MPI execution?

Reply:
Agreed. It would be good to have a note of which MPI and when this occurs, example good and bad output. Since this is a brittle fix and others like it could occur, perhaps move to a dedicated file/function of workarounds? Check for unexpected output and print a warning?

Reply:
This is the second time I've seen this behavior over the last 3 years.  The string in question is written to stderr whenever mpirun is executed and does not affect the correctness of what is run.  It is machine configuration dependent and appears to be related to OpenMPI.  This is essentially a quality of life fix that allows me to continue development unaffected by this apparently benign behavior.  In any event, the issuance of this string does not constitute an error in Nexus and so it is correct to guard against it.

Reply:
Whatever MPI library decide to print is out of our control. See https://github.com/QMCPACK/qmcpack/issues/4322. If I remember correctly, it is related to OpenMPI built with GPU support but there is no GPU on the compute node.

Reply:
@jtkrogel Thanks for the description - this is the kind of context I was requesting.

Searching this issue, it appears many people have seen this message with MPI, but there doesn't appear to be any definitive explanation of why it happens.   The MIT-MAGIC-COOKIE message is associated with X11 authentication, but several reports indicate it happens with no X11 visibly involved.

Could the comment be changed to something more precise, like "Handle spurious error message from OpenMPI" (and optionally link to these comments on the PR).

Reply:
Done